server/src/main/java/org/opensearch/Version.java

```diff
@@ -82,7 +82,8 @@
     public static final Version V_1_2_5 = new Version(1020599, org.apache.lucene.util.Version.LUCENE_8_10_1);
     public static final Version V_1_3_0 = new Version(1030099, org.apache.lucene.util.Version.LUCENE_8_10_1);
     public static final Version V_1_3_1 = new Version(1030199, org.apache.lucene.util.Version.LUCENE_8_10_1);
-    public static final Version CURRENT = V_1_3_1;
+    public static final Version V_1_3_2 = new Version(1030199, org.apache.lucene.util.Version.LUCENE_8_10_1);    
```

Comment:
This is supposed to be `new Version(1030299,...`?

Reply:
Changed it. Thanks!